apps/src/templates/codeReviewGroups/StudentGroup.jsx

```diff
@@ -20,18 +25,24 @@ export default function StudentGroup({droppableId, members}) {
       {(provided, snapshot) => (
         <div
           ref={provided.innerRef}
-          style={getListStyle(snapshot.isDraggingOver)}
+          style={{...getListStyle(snapshot.isDraggingOver), ...dropAreaStyle}}
           {...provided.droppableProps}
         >
-          {members.map((member, index) => (
-            <Student
-              followerId={member.followerId}
-              name={member.name}
-              index={index}
-              key={member.followerId}
-            />
-          ))}
-          {provided.placeholder}
+          {members && members.length > 0 ? (
+            <div style={styles.studentList}>
+              {members.map((member, index) => (
+                <Student
+                  followerId={member.followerId}
+                  name={member.name}
+                  index={index}
+                  key={member.followerId}
+                />
+              ))}
+              {provided.placeholder}
+            </div>
+          ) : (
+            showEmptyGroupPlaceholder && <div style={styles.emptyGroup} />
```

Comment:
I guess I'm a little confused by the two separate props, is it not possible in both cases for each scenario (`CodeReviewGroup` and `UnassignedStudentsPanel`) to pass in their own `emptyGroupStyle` or something like that?

Reply:
The two props are doing slightly different things here.`dropAreaStyle` is for the entire area to drop into, which for unassigned we want to be as large as the panel and not have a border. We want this to be the case for both empty and non-empty instances of the unassigned panel. `showEmptyGroupPlaceholder` is whether or not to have a div for an empty group or not.